include/villas/nodes/socket.hpp

```diff
@@ -22,7 +22,9 @@ class NodeCompat;
 
 struct Socket {
   int sd; // The socket descriptor
+  int clt_sd; // TCP client socket descriptor
   int verify_source; // Verify the source address of incoming packets against socket::remote.
+  bool tcp_connect = false; // TCP connection status bit
```

Comment:
The comment is not descriptive. What does this flag indicate? From the code, I can only guess that its used to keep track of the current connection state.

Can we please name this `tcp_connected` instead?